src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -57,17 +57,24 @@ case class BaggedSingleResult(
   override def getExpected(): Seq[Double] = Seq(expected)
 
   private lazy val expected = treePredictions.sum / treePredictions.length
-  private lazy val treeVariance: Double = treePredictions.map(x => Math.pow(x - expected, 2.0)).sum / treePredictions.length
+  private lazy val treeVariance: Double = {
+    assert(treePredictions.length > 1, "Bootstrap variance undefined for fewer than 2 bootstrap samples." + treePredictions.length)
+    treePredictions.map(x => Math.pow(x - expected, 2.0)).sum / treePredictions.length
```

Comment:
No bessel correction?

Reply:
I removed the Bessel correction because it was inconsistent with BaggedMultiResult and couples to the Jackknife in a way that will need to be analyzed more closely. I have opened an issue for this: https://github.com/CitrineInformatics/lolo/issues/213